projects/services/public-api-service-typescript/src/service.ts

```diff
@@ -43,6 +52,25 @@ export default class PublicApiHTTPService extends Service {
     const { httpPort } = (<PublicApiHTTPServiceConfig>this.config);
     this.server = this.app.listen(httpPort);
     logger.info(`Started HTTP public api on port ${httpPort}`);
+
+  }
+
+  // TODO Unify this code with GRPCServer.ts
+  private startManagementServer(startupCheckRunner: StartupCheckRunner): any {
+    logger.info("PublicApiService.startManagementServer starts");
+    const { httpManagementPort } = (<PublicApiHTTPServiceConfig>this.config);
+    const mgmtApp = express();
+    mgmtApp.get("/admin/startupCheck", (req: express.Request, res: express.Response) => {
+      startupCheckRunner.run()
+        .then((result: StartupStatus) => {
+          const httpCode = result.status === STARTUP_STATUS.OK ? 200 : 503;
+          return res.status(httpCode).send(result);
+        });
+    });
+
+    const mgmtServer = mgmtApp.listen(httpManagementPort);
+    logger.info(`Started HTTP public api management server on port ${httpManagementPort}`);
+    return mgmtServer;
```

Comment:
it checks that the public api 'business logic' is running, but can it also check if the express started okay and running? (meaning the service itself)

Reply:
Let's talk on Sunday about it. Maybe it's worth adding to the other services too.

Reply:
Added check that Public API HTTP Server is alive and listening.